### DISCUSSION TITLE
Nie dizla

### DISCUSSION THREAD

**Discussion by Shiver093** [Endiasz/Instaling-Bot#4] [General]
### Nie działa zapisywanie, jeśli masz czas i chcesz pomóc naprawdę dzięki.

**Comment by Endiasz** [comment]
Część. Testowałem to u siebie na czysto zainstalowanym rozszerzeniu i wszystko działało poprawie. 
Mam podejrzenie, twoja przeglądarka może usuwać zapisane dane, dane odnośnie słówek są zapisywane w lokalnej pamięci przeglądarki czy jakoś tak, jak korzystasz z opery to spróbuj na google chrome.


Jeżeli dalej występuje błąd spróbuj go bardziej sprecyzować.
